orca-pipelinetemplate/src/test/resources/gate.yml

```diff
@@ -0,0 +1,150 @@
+#document netflix-conventions
+clusters:
+  netflix-conventions:
+    requires:
+      membership:
+        - nf-datacenter
+        - nf-infrastructure
+
+accounts:
+  awsdefault:
+    cloudProvider: aws
+    regions:
+      us-east-1:
+        name: us-east-1 #baerf
+        zones:
+          - us-east-1c
+          - us-east-1d
+          - us-east-1e
+      us-west-2:
+        name: us-west-2  #backrefs?
+        zones:
+          - us-west-2a
+          - us-west-2b
+          - us-west-2c
+
+  mgmttest:
+    extends: awsdefault
+    accountId: 123456789012
+
+  mgmt:
+    extends: awsdefault
+    accountId: 234567890123
+
+
+---
+
+#document spinnaker-conventions
+
+environments:
+  spinBase:
+    regions:
+      us-west-2: {}
```

Comment:
an example of maps vs lists (in this case region actually potentially has useful values which warrant it being an object)